daemon/logger/copier.go

```diff
@@ -23,14 +23,16 @@ type Copier struct {
 	copyJobs  sync.WaitGroup
 	closeOnce sync.Once
 	closed    chan struct{}
+	timeNow   func() time.Time
 }
 
 // NewCopier creates a new Copier
-func NewCopier(srcs map[string]io.Reader, dst Logger) *Copier {
+func NewCopier(srcs map[string]io.Reader, dst Logger, timeNow func() time.Time) *Copier {
```

Comment:
Why pass in a fn instead of a `time.Location`?

Reply:
Yep! Good point